src/templates/layouts/healthCareLocalFacility/HealthServices/HealthServices.tsx

```diff
@@ -1,7 +1,3 @@
-/* istanbul ignore file */
-// Ignoring this until we get tests written; it's just really detailed
-// scaffolding for now.
-
```

Comment:
This made the code coverage bit in CI ignore this file. We don't want that anymore, now that we're adding comprehensive test coverage.